version.txt

```diff
@@ -1 +1 @@
-0.2.6
```

Comment:
It seems that we can still use 0.2.6 version

Reply:
pypi cannot replace release, so it's better to make a new release.